R/milestone.R

```diff
@@ -23,13 +23,18 @@
 #'   - `event` - Event indicator.
 #'   - `treatment` - Grouping variable.
 #' @param ms_time Milestone analysis time.
-#'
+#' @param test_type Method to build the test statistics.
+#' There are 2 options:
+#'   - `"native"`: a native approach by dividing the KM survival difference by its standard derivations,
```

Comment:
Is the approach "native" or "naive"? The string passed to `test_type` below is `"naive"`

Reply:
Probably a typo - should be "naive" cc @LittleBeannie